slices/libasan6.yaml

```diff
@@ -0,0 +1,15 @@
+package: libasan6
+
+essential:
+  - libasan6_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+    contents:
+      /usr/lib/*-linux-*/libasan.so.6*:
+  copyright:
+    essential:
+      - gcc-11-base_copyright
```

Comment:
libasan6 has a symlink from our docs to gcc-11-base. I think we need to add it to the content of this slice. Same goes for libasan8

```
  copyright:
    essential:
      - gcc-11-base_copyright
    contents:
      /usr/share/doc/libasan6:
```

Reply:
+1